src/securejoin.rs

```diff
@@ -261,7 +259,7 @@ async fn securejoin(context: &Context, qr: &str) -> ChatId {
                 "vc-request-with-auth"
             },
             get_qr_attr!(context, auth).to_string(),
-            Some(own_fingerprint),
+            own_fingerprint,
```

Comment:
This is now an option itself.  The old code was buggy because it could end up passing `Some(None)` here because of it's removed `.unwrap_or_default()` above.  Thankfully in practice `get_self_fingerprint()` never fails.